templates/optionalFeatures/mock-api/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "json-mock-api",
+  "version": "1.0.0",
+  "description": "A simple JSON mock API",
+  "main": "index.js",
+  "scripts": {
+    "mock:rest": "json-server --watch src/db.json",
+    "mock:graphql": "json-graphql-server src/db.js"
```

Comment:
On which port by default this servers are going to run? Can we change/define it here so that they don't conflict with apps

Reply:
To use a port other than default 3000, you can run yarn mock:rest --p <your port here>